### PR TITLE
issue 1

### PR DESCRIPTION
**Pull request by ShubhamRasal**
add index.html and contactme.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<center>
+<h1> welcome to Creators by heart ❤️  </h1>
+
```
Comment (creatorsbyheart) [inline index.html:3]: good work

File: contactme.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+
+<body bgcolor="violet" text="blue"> 
+<center>
+<h2>
+    <b> Shubham Dharma Rasal</b>
+</h2>
+
+<h3>
```
Comment (creatorsbyheart) [inline contactme.html:9]: dont like this changes.. why so much white spaces

Comment (ShubhamRasal) [reply contactme.html:9]: yes sure.. I will update this..

File: index.html (hunk 1 above)
Comment (ShubhamRasal) [reply index.html:3]: thank you so much

**Review by creatorsbyheart — APPROVED** [review]

**Pull request merged**